libnmstate/state.py

```diff
@@ -132,6 +188,20 @@ def verify_interfaces(self, other_state):
 
         self._assert_interfaces_equal(other_state)
 
+    def verify_routes(self, other_state):
+        self._clean_routes()
+        other_state._clean_routes()
+        for iface_name, routes in six.viewitems(self.iface_routes):
+            if routes != other_state.iface_routes.get(iface_name):
+                raise NmstateVerificationError(
+                    format_desired_current_state_diff(
+                        {
+                            Route.KEY: routes
+                        },
+                        {
+                            Route.KEY: other_state.iface_routes.get(iface_name)
```

Comment:
This should include all routes for all interfaces that are also mentioned in `self.routes`, should it not`.